app/views/contact/foi/new.html.erb

```diff
@@ -8,7 +8,12 @@
   <%= form_for :contact, url: contact_foi_path, as: :post, authenticity_token: false, html: { class: "contact-form" } do |f| %>
     <%= hidden_field_tag 'foi[url]', Plek.new.website_root + contact_foi_path %>
 
-    <input type="text" id="val" name="foi[val]" style="display:none"/>
+    <%# Honeypot field intended to reduce spam %>
+    <div class="val" aria-hidden="true">
+      <label for="val">Ignore if you're not a robot</label>
+      <input type="text" tabIndex="-1" autocomplete="off" id="val" name="foi[val]"/>
```

Comment:
`tabIndex` should be `tabindex`: https://www.w3.org/TR/html51/editing.html#the-tabindex-attribute